pallets/subtensor/tests/mock.rs

```diff
@@ -123,10 +123,11 @@ parameter_types! {
     pub const InitialBondsMovingAverage: u64 = 900_000;
     pub const InitialStakePruningMin: u16 = 0;
     pub const InitialFoundationDistribution: u64 = 0;
-    pub const InitialDefaultTake: u16 = 11_796; // 18% honest number.
+    pub const InitialDefaultTake: u16 = 32_767; // 50% for tests (18% honest number is used in production (see runtime))
```

Comment:
why use a different number in tests and runtime?

Reply:
yeah I would also prefer consistency here without a good reason not to have it